### DISCUSSION TITLE
lazy loading failed for package ‘sophistication’

### DISCUSSION THREAD

**Discussion by dt2229** [explosion/spaCy#10364] [Help: Installation] [answered]
Hello,

I am trying to install Kenneth Benoit, Kevin Munger, and Arthur Spirling's package Sophistication on a macOS Big Sur. 

`devtools::install_github("quanteda/spacyr", build_vignettes = FALSE)`
`library("spacyr")`
`spacy_install()`
`spacy_initialize()`
`devtools::install_github("kbenoit/sophistication")`

Upon the `devtools::install_github` command, I receive the following error in R studio's console: 
![image](https://user-images.githubusercontent.com/78564218/155338151-0789d305-620d-43bc-90ca-381221e09468.png)

I have tried the following in trying to resolve the error: 
![image](https://user-images.githubusercontent.com/78564218/155338786-a7381145-9ab0-45c7-a4f0-ef1b43d734ef.png)

My understanding is that there is a mismatch with the build-type of R, and the python version it was pointing to. I have arm-based R installed on my system, but it was pointing to x86 based python, that's why it threw that error. The solution is to either install an x86 version of R, or get R to point to arm-based python. I tried to resolve via the latter but to no success. 

Here are some potentially useful info on my environment/system: 
![image](https://user-images.githubusercontent.com/78564218/155348946-5f464d13-df8f-4c5b-bb45-91105a2d3042.png)

Thanks for any potential help!

**Comment by polm** [comment, marked as answer]
First, note we aren't involved in the development of spacyr and our ability to help you with it is limited. 

Second, please don't post screenshots of text - copy and paste it as text. 

You say that you didn't have success with arm-based Python but don't provide any errors or anything, so I'm not sure how to help you. I believe spaCy should just work on M1s even with ARM Python. [This somewhat old thread](https://github.com/explosion/spaCy/discussions/9397) might be helpful, or if not check some of the other threads with the `plat / m1` label. If none of those help and you can provide more information about what's not working we should be able to help you.
